app/root.tsx

```diff
@@ -105,6 +127,8 @@ export const loader = async ({ request }: LoaderFunctionArgs) => {
         boroughs,
         communityDistricts,
         cityCouncilDistricts: null,
+        agencies,
+        // projectTypes,
```

Comment:
Out of scope, don't need to leave in placeholders for future work.

Reply:
Removed.